pkg/controller/atlasproject/atlasproject_controller.go

```diff
@@ -201,6 +201,12 @@ func (r *AtlasProjectReconciler) Reconcile(context context.Context, req ctrl.Req
 		return reconcile.Result{Requeue: true}, nil
 	}
 
+	if result = ensureMaintenanceWindow(ctx, projectID, project); !result.IsOk() {
+		ctx.SetConditionTrue(status.MaintenanceWindowReadyType)
+		r.EventRecorder.Event(project, "Normal", string(status.MaintenanceWindowReadyType), "")
```

Comment:
Looks like this creates a "Normal" event only in case of a "not OK" result.

Reply:
Thank you for your feedback ! I've indeed noticed it this morning and it has been fixed in a commit
I am still reviewing my code to spot bugs and I will mark the PR as ready for review when I'm done

Reply:
Nice 👍